README.md

```diff
@@ -754,11 +754,12 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 ### Libraries
 
 - [vite-ssr](https://github.com/frandiox/vite-ssr) - Simplified Server Side Rendering with an SPA-like developer experience. ![react] ![vue3]
-- [vite-plugin-ssr](https://vite-plugin-ssr.com) - Like Nuxt/Next.js but as a do-one-thing-do-it-well plugin. ![react] ![vue3] ![vue2] ![svelte]
+- [Vike aka. vite-plugin-ssr](https://vite-plugin-ssr.com) - Like Nuxt/Next.js but as a do-one-thing-do-it-well plugin. ![react] ![vue3] ![vue2] ![svelte]
```

Comment:
?

Reply:
If you go to the link you'll see that vite-plugin-ssr has been renamed to Vike. Maybe the link should go directly to https://vike.dev/ since the old link doesn't directly forward yet?